### ISSUE TITLE
'TSBackgroundFetch/TSBackgroundFetch.h' file not found

### ISSUE THREAD

**Issue by josiahbryan** [transistorsoft/capacitor-background-fetch#8]
## Your Environment
* Plugin version: 0.0.6
* Platform: iOS
* Device OS version: 12.1
* Device manufacturer / model: iPhone 12
* XCode version: 12.4
* Capacitor info (`npx cap doctor`)
```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 3.2.4
  @capacitor/core: 3.2.4
  @capacitor/android: 3.2.4
  @capacitor/ios: 3.2.4

Installed Dependencies:

  @capacitor/cli: 3.1.2
  @capacitor/core: 3.1.2
  @capacitor/android: 3.1.2
  @capacitor/ios: 3.1.2

[success] iOS looking great! 👌
```

* Plugin config provided to `BackgroundFetch.configure`: N/A - Not using it personally, installing as a dep for the background geolocation plugin

## Expected Behavior

Build should work

## Actual Behavior

Build fails with error:
`/Users/josiahbryan/devel/rubber/client/node_modules/@transistorsoft/capacitor-background-fetch/ios/Plugin/BackgroundFetchModule.h:3:9: 'TSBackgroundFetch/TSBackgroundFetch.h' file not found`

## Steps to Reproduce

1. `npm i -S @transistorsoft/capacitor-background-fetch`
2. `npx cap sync`
3. Open XCode project
4. Build

## Context

Build the app

## Debug logs

![image](https://user-images.githubusercontent.com/4821548/136420130-f21bb26a-1155-4ea3-80a2-4c63db58e7d7.png)

**Comment by josiahbryan** [comment]
Note that the capacitor background geolocation plugin also was failing, so I removed it from the project so I could troubleshoot this issue without confusing the two. Once I get this working, I'll see if the background geo plugin works finally. However, I've tried everything I can (googling the heck out of the error) and nothing I can find works. 

* I've tried to go into `ios/App` and `rm -rf Pods Podfile.lock` and then `pod install` - no joy. 
* Also tried to clean the build folder and rebuild - no joy. 
* I've removed the plugin and re-installed it - no joy.

Any thoughts on how to fix..?
